packages/mdc-drawer/component.ts

```diff
@@ -34,13 +34,18 @@ import {MDCDrawerFocusTrapFactory} from './util';
 
 const {cssClasses, strings} = MDCDismissibleDrawerFoundation;
 
+/**
+ * @events MDCDrawer:closed %-% Emits when the navigation drawer has closed. %-% {}
+ * @events MDCDrawer:opened %-% Emits when the navigation drawer has opened. %-% {}
```

Comment:
Follow-up with previous comment.

if `@event` is not supported by documentalist is it possible to still parse this with `@event` as custom annotation tag?

Also, this doesn't have to be a table, it can be a bullet list. I'm sure people surely forget to add this separator for future class names.

Reply:
Yeah I was thinking we could make it a list (originally i had it that way). 

Also the `@event` tag is being eaten by documentalist...so I had to create this fake tag. What if we use `@fires`?

Reply:
`@fires` doesn't seem to be valid annotation either. is `@fires` supported in documentalist?

Let's keep `@events` for now.

Reply:
ok sounds good - @fires is supported https://jsdoc.app/tags-fires.html